### ISSUE TITLE
Auto-correct ParserBlockingScript check with defer

### ISSUE THREAD

**Issue by charlespwd** [Shopify/theme-check#239]
Couple of ways we could tackle this but I don't think the automatic fix is a good way. Since this _might_ break things. But we should have a "fixer" for it anyway. Something we could do from the editor with a code action maybe?

**Comment by macournoyer** [comment]
You mean using the existing autocorrect feature? => https://github.com/Shopify/theme-check/blob/master/lib/theme_check/checks/space_inside_braces.rb#L51

And this feature could be exposed to vscode as a quick action?

**Comment by charlespwd** [comment]
Yeah the thing is though that if you use -a I don't think we should apply it. Its application is not really something you can do blindly because it does affect the output in a way. 

And I believe the quick actions are named code actions in the LSP :P So yes!
